signing_service/settings/base.py

```diff
@@ -225,7 +225,7 @@
 
 # When True, it means Hawk authentication will be disabled everywhere.
 # This is mainly just to get a speed-up while testing.
-SKIP_HAWK_AUTH = False
+SKIP_HAWK_AUTH = True
```

Comment:
@rtilder is planning on using Hawk rather that OAuth for this bug: https://bugzilla.mozilla.org/show_bug.cgi?id=956363, first he wants to get the new server in and working, then start turning on Hawk. But that requires zamboni chnages to support Hawk.